csv/parse_stream_test.ts

```diff
@@ -341,18 +394,21 @@ x,,,
         if ("comment" in testCase) {
           options.comment = testCase.comment;
         }
-        if ("skipFirstRow" in testCase) {
-          options.skipFirstRow = testCase.skipFirstRow;
-        }
-        if ("columns" in testCase) {
-          options.columns = testCase.columns;
-        }
         if ("trimLeadingSpace" in testCase) {
           options.trimLeadingSpace = testCase.trimLeadingSpace;
         }
         if ("lazyQuotes" in testCase) {
           options.lazyQuotes = testCase.lazyQuotes;
         }
+        if ("fieldsPerRecord" in testCase) {
+          options.fieldsPerRecord = testCase.fieldsPerRecord;
+        }
+        if ("skipFirstRow" in testCase) {
+          options.skipFirstRow = testCase.skipFirstRow;
+        }
+        if ("columns" in testCase) {
+          options.columns = testCase.columns;
+        }
```

Comment:
This adds `fieldsPerRecord`, and also reorders `skipFirstRow` and `columns` to match the order of properties declared in `CsvParseStreamOptions` so that it will be easy for us to notice the missing field in the future